addon/components/modal-dialog.js

```diff
@@ -14,111 +22,60 @@ const computedJoin = function(prop) {
 };
 
 export default Ember.Component.extend({
-  // modal-dialog is itself tagless. positioned-container provides the container div
   tagName: '',
-
-  layout: template,
-  modalService: injectService('modal-dialog'),
-  destinationElementId: reads('modalService.destinationElementId'),
-
-  useEmberTether: computed('modalService.useEmberTether', 'alignment', 'renderInPlace', function() {
-    return this.get('modalService.useEmberTether') && (this.get('alignment') !== 'none') && !this.get('renderInPlace');
+  layoutStrategy: computed('structure', 'useEmberTether', function() {
+    let strategy = get(this, 'structure');
+    const useEmberTether = get(this, 'useEmberTether');
+    if (strategy === 'tethered' && !useEmberTether) {
```

Comment:
I think it's better to have a stable default option, that doesn't rely on a dependency. The DOCS etc. can still highlight tether as the suggested mode, if you wish.

Similar to how e.g. Rails ships with a default job-handler thats very basic (and people are expected to use something else).